libfswatch/CMakeLists.txt

```diff
@@ -179,14 +179,17 @@ if (USE_NLS)
     endif ()
 endif ()
 
-target_include_directories(libfswatch PUBLIC src/libfswatch)
-target_include_directories(libfswatch PUBLIC src)
+target_include_directories(libfswatch PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/src>
+    $<INSTALL_INTERFACE:include>)
 target_include_directories(libfswatch PUBLIC ${Intl_INCLUDE_DIRS})
 target_include_directories(libfswatch PRIVATE ${PROJECT_BINARY_DIR})
-target_include_directories(libfswatch PRIVATE ${CMAKE_CURRENT_BINARY_DIR})
 target_link_libraries(libfswatch PRIVATE ${EXTRA_LIBS})
 
-install(TARGETS libfswatch LIBRARY DESTINATION lib)
+install(TARGETS libfswatch EXPORT libfswatch-export LIBRARY DESTINATION lib)
+install(EXPORT libfswatch-export DESTINATION lib/cmake/libfswatch
```

Comment:
This should be:

```
install(EXPORT libfswatch-export DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/libfswatch
```

As it needs to be installed to /usr/lib64 on 64bit platforms.


I guess that other places needs to be adjusted too. You need `include(GNUInstallDirs)` to get access to the variables.

Reply:
Thanks, I'll fix it if @emcrisostomo shows up.